lib/mkmf.rb

```diff
@@ -679,19 +679,19 @@ def with_cppflags(flags)
   end
 
   # :nodoc:
-  def try_cppflags(flags, opts = {})
-    try_header(MAIN_DOES_NOTHING, flags, {:werror => true}.update(opts))
+  def try_cppflags(flags, werror: true, **opts)
+    try_header(MAIN_DOES_NOTHING, flags, werror: werror, **opts)
```

Comment:
```suggestion
    try_header(MAIN_DOES_NOTHING, flags, werror:, **opts)
```

Reply:
mkmf.rb will be used also by BASERUBY, which may not support forwarding keyword arguments.